test/mocks/data/firestore_user.dart

```diff
@@ -42,3 +42,13 @@ class FirestoreUserGenerator {
     }).toList();
   }
 }
+
+Future<void> setUserFirestore(
+  FirebaseFirestore firestore,
+  UserFirestore user,
+) async {
+  await firestore
+      .collection(UserFirestore.collectionName)
+      .doc(user.uid.value)
+      .set(user.data.toDbData());
+}
```

Comment:
I added the same thing in my PR #190. I think this is fine, we'll just have to be careful while merging.